.github/workflows/build.yaml

```diff
@@ -23,7 +23,7 @@ jobs:
       contents: read
       packages: write
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v3
```

Comment:
upversion the plugin, to address this warning: https://github.com/actions/checkout/issues/1047